inception/inception-api-annotation-experimental/src/main/ts/client/messages/request/NewViewportRequest.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Viewport} from "../../model/Viewport";
+
+export class NewViewportRequest {
```

Comment:
We seem to have some "create..." and some "new..." requests. Maybe call them all "create..."?

Reply:
I thought it might be a bit misleading when Viewport - and Document have a "create" in the beginning, as the client simply requests a new document / new viewport with whom he wants to work with, instead of really want the server to "create" one. 
But rather when the client "createSpanRequest" he really wants the server to create a span.